### PR TITLE
created 3 new user groups cityadmin group civilian all civilian 

### PR DESCRIPTION
**Pull request by jagrutimohanty**
With the amplify cli created 3 groups - cityadmin , group civilian and all the civilian group. Plan is to add the users in these groups and test with activity apli to start with

### CODE REVIEW

**Comment by jagrutimohanty** [comment]
Could you please approve

File: amplify/backend/auth/covidcounterba0a32fb/parameters.json [hunk 1]
```diff
@@ -45,16 +45,12 @@
...
     },
-    "useDefault": "defaultSocial",
+    "useDefault": "default",
     "usernameAttributes": [
         "email"
     ],
     "userPoolGroupList": [],
     "serviceName": "Cognito",
     "dependsOn": [],
-    "hostedUI": true,
```
Comment (shannonphu) [inline amplify/backend/auth/covidcounterba0a32fb/parameters.json:55]: This should not have been removed

File: amplify/backend/auth/covidcounterba0a32fb/parameters.json [hunk 2]
```diff
@@ -45,16 +45,12 @@
             "Arn"
         ]
     },
-    "useDefault": "defaultSocial",
```
Comment (shannonphu) [inline amplify/backend/auth/covidcounterba0a32fb/parameters.json:48]: This should stay as defaultSocial i think

File: amplify/backend/auth/covidcounterba0a32fb/parameters.json (hunk 1 above)
Comment (jagrutimohanty) [reply amplify/backend/auth/covidcounterba0a32fb/parameters.json:55]: Hey Shannon this code is present currently , comma has been removed by new changes

File: amplify/backend/auth/covidcounterba0a32fb/parameters.json (hunk 2 above)
Comment (jagrutimohanty) [reply amplify/backend/auth/covidcounterba0a32fb/parameters.json:48]: sure Shannon I will add defaultSocial instead of default

**Pull request merged**